plenum/server/node.py

```diff
@@ -2042,7 +2055,14 @@ def processRequest(self, request: Request, frm: str):
     def is_query(self, txn_type) -> bool:
         # Does the transaction type correspond to a read?
         handler = self.get_req_handler(txn_type=txn_type)
-        return handler and handler.is_query(txn_type)
+        return handler and isinstance(handler,
+                                      LedgerRequestHandler
+                                      ) and handler.is_query(txn_type)
+
+    def is_action(self, txn_type) -> bool:
+        handler = self.get_req_handler(txn_type=txn_type)
+        return handler and (
+            txn_type in self.actionReqHandler.operation_types)
```

Comment:
What is the reason to retrieve `handler`?

Reply:
No reason. Will fix it.